src/app/components/core/configuration-manager/view-config-item/view-config-item.component.css

```diff
@@ -87,6 +87,10 @@
   cursor: pointer !important;
 }
 
+.codemirror-field {
+  width: 610px;
```

Comment:
Tested on different browsers?

Reply:
Yes, I've tested it on Safari and Chrome browser. It is working fine on both.